js/src/wasm/WasmBaselineCompile.cpp

```diff
@@ -2264,12 +2264,10 @@ class BaseStackFrame final : public BaseStackFrameAllocator {
   }
 
   void storeImmediateF32ToStack(float imm, uint32_t destHeight, Register temp) {
-    union StoredFloat32 {
-      int32_t i32;
+    union {
       float f32;
-
-      StoredFloat32(float f32) : f32(f32) {}
-    } bits{imm};
+      int32_t i32;
+    } bits{imm};  // Note: the default constructor uses the first (f32) member.
```

Comment:
```suggestion
    } bits{imm};
```
The note is incorrect as this isn't a default constructor.
Could fix it but I think maybe we don't need a note for the language feature?

Reply:
I deleted the comment.